components/HtmlHead.tsx

```diff
@@ -14,9 +14,15 @@ const HtmlHead = ({ frontMatter }: HeaderProps) => {
   const pageTitle = frontMatter.title || siteConfig.title;
   const canonicalLink = `https://nodejs.org${route}`;
 
+  // "{pageTitle} | {siteConfig.title}" if pageTitle is defined
+  const title =
+    // avoid "Node.js | Node.js" title
+    pageTitle && pageTitle !== siteConfig.title
```

Comment:
📝 Some page's `${pageTitle}` is "Node.js" and I avoid "Node.js | Node.js".

- `http://localhost:3000/en/`: `{ pageTitle: 'Node.js', siteConfigTitle: 'Node.js' }`
- `http://localhost:3000/en/blog/`: `{ pageTitle: 'Node.js', siteConfigTitle: 'Node.js' }`